### DISCUSSION TITLE
Cannot see the collector.

### DISCUSSION THREAD

**Discussion by hgati** [netdata/netdata#15799] [Q&A] [answered]
https://learn.netdata.cloud/docs/configuring/#enabledisable-specific-collectors

According to the description above, various service configuration files should be located under the go.d directory for different services. However, there is nothing on my server. Nevertheless, netdata still displays statistics for various services properly.

The configuration files go.d.conf and go.d/mysql.conf are both missing. Is this normal?

![](https://i.ibb.co/qnt33Zp/009.png)

**Comment by ilyam8** [comment, marked as answer]
Hi, @hgati. They are not missing, it is expected to have no user configuration files in the `/etc/netdata` - see [Redis readme](https://learn.netdata.cloud/docs/data-collection/database-servers/redis#configuration).
